### ISSUE TITLE
ActionMenu: Add border for accessibility

### ISSUE THREAD

**Issue by TravisBumgarner** [pluralsight/design-system#363]
### Expected behavior

Each Button for an ActionMenu inside a DropDown should have the border (blue in Chrome) for accessibility.

### Actual behavior

There is a very faint gray and a slightly darker gray that distinguish the two now. 
<img width="181" alt="screen shot 2018-11-27 at 3 56 21 pm" src="https://user-images.githubusercontent.com/11890057/49111036-02f08e80-f25d-11e8-88ad-b8d0edf5d82b.png">

**Comment by dborodyansky** [comment]
I think we may need a custom indicator (attached) for keyboard focus here in action menus, rather than focus `halo`. What do you think @jaketrent @danethurber ?

![image](https://user-images.githubusercontent.com/3674406/50117433-d5668780-0201-11e9-8691-1cfa5d10d00f.png)

**Comment by danethurber** [comment]
@dborodyansky i think you're right. i like the middle and last one

**Comment by jaketrent** [comment]
@dborodyansky num 3 is fav, then num 2, then num 1

**Comment by jaketrent** [comment]
Triaged: using style num 3 for keyboard focus only; keeping mouse hover same.
